ambry-store/src/test/java/com.github.ambry.store/BlobStoreTest.java

```diff
@@ -1180,6 +1181,66 @@ public void storeIoErrorCountTest() throws StoreException, IOException {
     reloadStore();
   }
 
+  /**
+   * Test both success and failure cases when deleting store files.
+   * @throws Exception
+   */
+  @Test
+  public void deleteStoreFilesTest() throws Exception {
+    assumeTrue(isLogSegmented);
```

Comment:
Could this code be tested with non segmented logs? Perhaps just put the test of swap segments within `if (isLogSegmented)`

Reply:
Good point. I added some codes to ensure both segmented and non-segmented cases are tests.